usr/share/rear/lib/_input-output-functions.sh

```diff
@@ -403,6 +403,14 @@ function UserInput () {
     local timeout=300
     # Avoid stderr if USER_INPUT_TIMEOUT is not set or empty and ignore wrong USER_INPUT_TIMEOUT:
     test "$USER_INPUT_TIMEOUT" -ge 0 2>/dev/null && timeout=$USER_INPUT_TIMEOUT
+    # Have some seconds (at least one second) delay when an automated user input is used to be fail-safe against
+    # a possibly false specified predefined user input value for an endless retrying loop of UserInput calls
+    # that would (without the delay) run in a tight loop that wastes resources (CPU, diskspace, and memory)
+    # and fills up the ReaR log file (and the disk - which is a ramdisk for 'rear recover')
+    # with some KiB data each second that may let 'rear recover' fail with 'out of diskspace/memory':
+    local automated_input_interrupt_timeout=3
+    # Avoid stderr if USER_INPUT_INTERRUPT_TIMEOUT is not set or empty and ignore wrong USER_INPUT_INTERRUPT_TIMEOUT:
+    test "$USER_INPUT_INTERRUPT_TIMEOUT" -ge 1 2>/dev/null && automated_input_interrupt_timeout=$USER_INPUT_INTERRUPT_TIMEOUT
```

Comment:
All this will happen for ReaR v 2.3 - not now, cf.
https://github.com/rear/rear/issues/1399

Because ReaR v 2.2 was preponed I will not have
things officially documented in default.conf
that are not fully ready for our users
from my current point of view.

Currenzly I want to be free to change it as I like
until everything looks good from my point of view
and then I will make it "official" in default.conf.